File: Trivia.NET/QuizQuestion/MultipleChoiceQuestion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Trivia.NET.QuizQuestion
{
    public class MultipleChoiceQuestion : Question
    {
        public string Correct { get; set; }

        public MultipleChoiceQuestion(string query)
        {
            Type = QuestionType.MultipleChoice;
            Query = query;
            Answer = "";
        }
        public MultipleChoiceQuestion(string query, string[] answers) : this(query)
        {
            Answers = answers;
        }

        public override void DisplayAnswers()
        {
            int letter = 64; // ASCI @
            foreach(string item in Answers)
            {
                letter++;
                Console.WriteLine($"{ (char)letter }: { item }");
            }
        }
        public void GetCorrectAnswer(string input)
        {
            List<string> newAnswers = Answers.ToList();
            newAnswers.RemoveAt(Answers.Length - 1);
            Answers = newAnswers.ToArray();

            Correct = Answers[input.ToUpper()[0] - '@' - 1];
        }
        public override bool CheckAnswer(string input)
        {
            return base.CheckAnswer(Answers[input.ToUpper()[0] - '@' - 1]);
        }
    }
}

File: NodeTrivia/src/trivia.net/quizQuestion/multipleChoiceQuestion.ts
namespace QuizQuestion {
    export class MultipleChoiceQuestion extends Question {
        correct: string;

        constructor(query: string, answers: string[]) {
            super(QuestionType.MultipleChoice, query, answers);
        }

        displayAnswers(): any {
            let letter: number = 64; // @ in ASCII
            let display: string;
            this.answers.forEach(function(item: string) {
                letter++;
                display = `${ String.fromCharCode(letter) }: ${ item }`;
            });
            return display;
        }

        getCorrectAnswer(input: string): void
        {
            let newAnswers: string[] = this.answers;
            newAnswers.splice(this.answers.length - 1);
            this.answers = newAnswers;

            this.correct = this.answers[input.toUpperCase().charAt(0).charCodeAt(0) - 93];
        }
        checkAnswer(input: string): boolean
        {
            return super.checkAnswer(this.answers[input.toUpperCase().charAt(0).charCodeAt(0) - 93]);
        }
    }
}
